compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2578,6 +2581,18 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
         }
     }
 
+    /// `#[rustc_non_const_sized]` is a testing attribute for forcing `const Sized` not to be
+    /// implemented (while there are no types which are non-`const Sized`) - it should only be
+    /// applied to structs or enums.
```

Comment:
"forcing `const Sized` not to be implemented while there are no types which are non-`const Sized`"...? ...forgive me, I'm not sure I understand, do you mean disabling the assumption the type would be `const Sized`, or do you mean actively requiring it to be `!const Sized`?

Reply:
Rewritten this comment, should be clearer